nibabel/streamlines/tests/test_tractogram.py

```diff
@@ -593,21 +602,21 @@ def test_tractogram_copy(self):
         tractogram = DATA['tractogram'].copy()
 
         # Check we copied the data and not simply created new references.
-        assert_true(tractogram is not DATA['tractogram'])
-        assert_true(tractogram.streamlines
-                    is not DATA['tractogram'].streamlines)
-        assert_true(tractogram.data_per_streamline
-                    is not DATA['tractogram'].data_per_streamline)
-        assert_true(tractogram.data_per_point
-                    is not DATA['tractogram'].data_per_point)
+        assert tractogram is not DATA['tractogram']
+        assert tractogram.streamlines is \
+                not DATA['tractogram'].streamlines
+        assert tractogram.data_per_streamline is \
+                not DATA['tractogram'].data_per_streamline
+        assert tractogram.data_per_point is \
+                not DATA['tractogram'].data_per_point
 
         for key in tractogram.data_per_streamline:
-            assert_true(tractogram.data_per_streamline[key]
-                        is not DATA['tractogram'].data_per_streamline[key])
+            assert tractogram.data_per_streamline[key] is \
+                not DATA['tractogram'].data_per_streamline[key]
 
         for key in tractogram.data_per_point:
-            assert_true(tractogram.data_per_point[key]
-                        is not DATA['tractogram'].data_per_point[key])
+            assert tractogram.data_per_point[key] is \
+                not DATA['tractogram'].data_per_point[key]
```

Comment:
And these next two.

The general line limit is 100 characters, but we don't even enforce that in tests.